kvm-ioctls/coverage_config_aarch64.json

```diff
@@ -1,5 +0,0 @@
-{
-  "coverage_score": 77.1,
```

Comment:
I would prefer to have an independent coverage test for the kvm-ioctls. For this we would need to add a custom test that runs pytest from the directory I think. If it's too complicated, we can also add a separate issue to handle it.

Reply:
Noted, I will work it out :)

Reply:
I just recollected if we were to do coverage test per crate, we'd better update our CI to utilize codecov. I will try to do it first and come back.

How about we leave this as it is for now :)

Reply:
The current CI supports running individual coverage tests, no need to switch to codecov for this reason. But maybe codecov has some other features that make it a better choice. In any case, we can just open an issue and address this later. No need to spend time on it now.